### PR TITLE
Xcode 8.3 updates

### PR DESCRIPTION
**Pull request by bg-stripe**
r? @bdorfman-stripe

### CODE REVIEW

File: Stripe/Info.plist [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>Stripe</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.stripe.stripe-ios</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```
Comment (bdorfman-stripe) [inline Stripe/Info.plist]: Is this PRODUCT_BUNDLE_IDENTIFIER set correctly somewhere else?

Comment (bg-stripe) [inline Stripe/Info.plist]: yep – it's in our project xcconfig: https://github.com/stripe/stripe-ios/blob/a26ff49ddfbcee9b1cc7e82b1ecfdd66794d09f4/Stripe/BuildConfigurations/Project-Shared.xcconfig#L139

**Comment by bdorfman-stripe** [comment]
LGTM!

File: Stripe/Info.plist (hunk 1 above)
Comment (bg-stripe) [inline Stripe/Info.plist]: actually, looks like this wasn't working – our manual installation test was catching the issue. i looked into it for a bit, but timed out and decided to just revert the change, and remove the unused PRODUCT_BUNDLE_IDENTIFIER from the xcconfig.

**Comment by bdorfman-stripe** [comment]
Huh, ok either way works i guess
